### PR TITLE
crate.md update

### PR DESCRIPTION
**Pull request by Flavourized**
* Fixed itemIDs -> materials
* Fixed markdown
* Fixed grammar

### CODE REVIEW

File: docs/config/crate.md [hunk 1]
```diff
@@ -9,56 +10,65 @@
...
 * Unlimited crates and rewards
 * Multi-world support
-* Scrolling Animations
+* Animations
 * Preview Menu
-* Messaging
+* Message System
 * Particle Effects
 * Rewards support items and commands
+<br></br>
```
Comment (imWillX) [inline docs/config/crate.md]: Could we remove the line breaks? 

Our theme will handle these!

File: docs/config/crate.md [hunk 2]
```diff
@@ -9,56 +10,65 @@
...
+### Crate Types
 
-### Type
-
-Types define the unique characteristic and interactions that activate a crate. These are the following types...
+The type of crate defines the unique interaction with the player when the crate is activated. These are the following types..
```
Comment (imWillX) [inline docs/config/crate.md]: one more dot for ocd

File: docs/config/crate.md [hunk 3]
```diff
@@ -9,56 +10,65 @@
...
 | -------- | --------------------- | ------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------- |
 | SUPPLY   | No                    | Placeable crate that acts as a minecraft chest                                        | Placing down a chest                                                                                                            |
 | MYSTERY  | Yes                   | Crate that is activated by any type of click                                          | Right or left click with the crate in-hand                                                                                      |
 | KEY      | Yes                   | Crate that is preset to a block. This block acts as a hub for users to interact with. | Right or left click a preset block with the crate in-hand. Left click is set to preview while right click is to open the crate. |
 
+<br></br>
```
Comment (imWillX) [inline docs/config/crate.md]: remove pls, thx

**Comment by Flavourized** [comment]
I HELPED 😅

File: docs/config/crate.md [hunk 4]
```diff
@@ -9,56 +10,65 @@
...
 ### The Physical Crate or Key
 
-Crates and keys are physical items in Minecraft. These physical items are what players use to interact and activate rewards.
-Read more about the [item parsing](#item-parser)
+Crates and keys are physical items in Minecraft. These physical items are what players use to interact and activate rewards. Each crate or key-crate has its own item or key, respectively. 
+
+
```
Comment (imWillX) [inline docs/config/crate.md]: extra line

File: docs/config/crate.md [hunk 5]
```diff
@@ -9,56 +10,65 @@
...
 ```YML
 FoodKey:
     ...
-    item: '131 1 name:&c[&6Foodie_&eKey&c]_&f#124 lore:&7Mmmm,_something_smells_good.\n&8Place_the_chest_down_to_obtain_a_pack_of_food! glow:true'
+    item: 'tripwire_hook 1 name:&c[&6Foodie_&eKey&c]_&f#124 lore:&7Mmmm,_something_smells_good.\n&8Place_the_chest_down_to_obtain_a_pack_of_food! glow:true'
     ...
 ```
 
-### Animation
+><span style="color:red">**NOTE:** </span>This plugin uses the EssentialsX item format. For more information about item parsing,  [click here](#item-parser).
```
Comment (imWillX) [inline docs/config/crate.md]: Use '>' to specify a green note and "!>" to specify a red note.
```
> Green note
!> This plugin uses the EssentialsX item format. For more information about item parsing,  [click here](#item-parser).
```
See https://hazebyte.github.io/CrateReloaded/#/reference/particles

File: docs/config/crate.md [hunk 6]
```diff
@@ -9,56 +10,65 @@
...
+
+<br></br>
```
Comment (imWillX) [inline docs/config/crate.md]: remove pls

File: docs/config/crate.md [hunk 7]
```diff
@@ -9,56 +10,65 @@
...
+| ------------- | ---------- |
+| ROULETTE      | BLANK      |
+| CSGO          | RANDOM     |
+| REVERSE_CSGO  |            |
+| WHEEL         |            |
+| REVERSE_WHEEL |            |
 
+</table>
+
+<br></br>
```
Comment (imWillX) [inline docs/config/crate.md]: remove line break pls.

File: docs/config/crate.md [hunk 8]
```diff
@@ -69,9 +79,14 @@ FoodKeyT2:
     ...
 ```
 
+
+><span style="color:red">**NOTE:** </span>For holograms to work, you must have [HolographicDisplays](https://dev.bukkit.org/projects/holographic-displays) installed on your server
```
Comment (imWillX) [inline docs/config/crate.md]: Use '!>' or '>'

File: docs/config/crate.md [hunk 9]
```diff
@@ -69,9 +79,14 @@ FoodKeyT2:
...
+
+
+<br></br>
```
Comment (imWillX) [inline docs/config/crate.md]: remove Line Break pls.

**Pull request merged**